src/App.js

```diff
@@ -51,6 +57,20 @@ function App() {
     completePsiturk();
   };
 
+  // Function to add jspsych data on login
+  const setLoggedIn = useCallback(
```

Comment:
what's the reasoning around `useCallback` vs a plain function? (just curious, I'm not familiar with it)